openedx/openedx-platform
# feat: TNL-11173 Authoring API is v0 for now

## Description

This ticket contains work from the TNL-11173 JIRA task.

The changes made here move existing authoring API capabilities from their former v1-based routes to v0-routes. This is to emphasize that the current API is experimental and expected to be deprecated in short order. It's also consistent with othern TNL work exposing public APIs under v0-based 'authoring' routes on our AWS API Gateway.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

Use the same testing procedures that applied when the studio API routes were initially created. The only change that applies is that the routes are now v0 routes instead of v1 routes. Continuing to make requests off of the v1 routes should result in 404 (not found) errors.

## Deadline

This PR is essential for carrying out the end-to-end tests we need to verify that our API Gateway configuration is actually working. It is therefore time sensitive.

## Other information
